ext/curl/curl.stub.php

```diff
@@ -3753,6 +3762,9 @@ function curl_share_strerror(int $error_code): ?string {}
 /** @refcount 1 */
 function curl_strerror(int $error_code): ?string {}
 
+/** @refcount 1 */
+function curl_persistent_share_init(array $share_options): CurlPersistentShareHandle {}
```

Comment:
Would name this either `curl_share_persistent_init` or `curl_share_init_persistent` to have the common prefix `curl_share` for easier discoverability (e.g. in autocompletion).

Reply:
I originally went with `curl_share_init_persistent`, as I did in the first PR, but decided on `curl_persistent_share_init` since it returned a `CurlPersistentShareHandle`, and not a `CurlSharePersistentHandle`. That said, maybe I could go with `CurlSharePersistentHandle`, since that would also promote discoverability?

Reply:
I think it's fine if the class name does not exactly match the function name. Regarding the ordering in the class name I don't have a strong opinion. I think you don't usually interact with the class directly, as you would simply pass along the handle via variables.